src/main/java/edu/harvard/dbmi/avillach/dictionary/util/JsonBlobParser.java

```diff
@@ -62,4 +72,17 @@ public Float parseMax(String valuesArr) {
         return parseFromIndex(valuesArr, 1);
     }
 
+    public Map<String, String> parseMetaData(String jsonMetaData) {
+        Map<String, String> metadata;
+
+        try {
+            List<Map<String, String>> maps = objectMapper.readValue(jsonMetaData, new TypeReference<List<Map<String, String>>>() {});
+            // convert the list to a flat map
+            metadata = maps.stream().collect(Collectors.toMap(entry -> entry.get("key"), entry -> entry.get("value")));
```

Comment:
I think this gives weird errors if you have duplicate keys.

Reply:
Should we ever have duplicate keys in the meta data for a given consent_node? Can you provide an example where this would happen?